Guevados/jamMetroidvania
# Improvement/pj 101

# Problem

We need to show the player initial state

# Solution 

Create Player handler class

# How to test

Download branch and run... Animated player should be show on center of the screen 


https://github.com/Guevados/jamMetroidvania/assets/28446213/8646d136-b0ee-4af9-be7f-530e929d49f0
